ios/FluentUI/Command Bar/CommandBarTokens.swift

```diff
@@ -0,0 +1,52 @@
+//
+//  Copyright (c) Microsoft Corporation. All rights reserved.
+//  Licensed under the MIT License.
+//
+
+import UIKit
+
+class MSFCommandBarTokens: MSFTokensBase, ObservableObject {
+    @Published public var defaultBackgroundColor: UIColor!
+    @Published public var defaultIconColor: UIColor!
+
+    @Published public var pressedBackgroundColor: UIColor!
+    @Published public var pressedIconColor: UIColor!
+
+    @Published public var dismissBackgroundColor: UIColor!
+    @Published public var dismissIconColor: UIColor!
+    @Published public var groupBorderRadius: CGFloat!
+    @Published public var groupInterspace: CGFloat!
+    @Published public var iconSize: CGFloat!
+    @Published public var itemInterspace: CGFloat!
+
+    /// Notifies the drawer controller to refresh its UI to reflect its design token values
+    var themeDidUpdate: (() -> Void)?
+
+    override init() {
+        super.init()
+
+        self.themeAware = true
+
+        updateForCurrentTheme()
+    }
+
+    @objc open func didChangeAppearanceProxy() {
+        updateForCurrentTheme()
+    }
+
+    override func updateForCurrentTheme() {
+        let currentTheme = theme
+        let appearanceProxy = currentTheme.MSFCommandBarTokens
+
+        defaultBackgroundColor = appearanceProxy.backgroundColor.rest
```

Comment:
Why are you naming these properties "default" instead of just backgroundColor and iconColor?